src/main/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsView.java

```diff
@@ -0,0 +1,543 @@
+
+    @FXML
+    private TableView<AbbreviationViewModel> journalAbbreviationsTable;
+    @FXML
+    private TableColumn<AbbreviationViewModel, String> journalTableNameColumn;
+    @FXML
+    private TableColumn<AbbreviationViewModel, String> journalTableAbbreviationColumn;
+    @FXML
+    private TableColumn<AbbreviationViewModel, Boolean> journalTableEditColumn;
+    @FXML
+    private TableColumn<AbbreviationViewModel, Boolean> journalTableDeleteColumn;
+    @FXML
+    private Button cancelButton;
+    @FXML
+    private Button saveButton;
+    @FXML
+    private ComboBox<AbbreviationsFile> journalFilesBox;
+    @FXML
+    private Button addJournalFileButton;
+    @FXML
+    private Button addNewJournalFileButton;
+    @FXML
+    private Button removeJournalAbbreviationsButton;
+    @FXML
+    private ProgressIndicator progressIndicator;
+    @FXML
+    private Label loadingLabel;
+
+
+    public ManageJournalAbbreviationsView() {
+        super();
+        bundle = Localization.getMessages();
+    }
+
+    @FXML
+    private void initialize() {
+        setUpTable();
+        setBindings();
+        setButtonStyles();
+        new Thread(new Runnable() {
+
+            @Override
+            public void run() {
+                loadingLabel.setVisible(true);
+                progressIndicator.setVisible(true);
+                viewModel.createFileObjects();
+                journalFilesBox.getSelectionModel().selectLast();
+                viewModel.addBuiltInLists();
+                loadingLabel.setVisible(false);
+                progressIndicator.setVisible(false);
+            }
+        }).start();
+    }
+
+    private void setButtonStyles() {
+        journalFilesBox.setPromptText(Localization.lang("No abbreviation files loaded"));
+        Text addJournalFileButtonGraphic = new Text(IconTheme.JabRefIcon.OPEN.getCode());
+        addJournalFileButtonGraphic.getStyleClass().add("icon");
+        addJournalFileButton.setGraphic(addJournalFileButtonGraphic);
+        Text addNewJournalFileButtonGraphic = new Text(IconTheme.JabRefIcon.NEW.getCode());
+        addNewJournalFileButtonGraphic.getStyleClass().add("icon");
+        addNewJournalFileButton.setGraphic(addNewJournalFileButtonGraphic);
+        Text removeJournalAbbreviationsButtonGraphic = new Text(IconTheme.JabRefIcon.CLOSE.getCode());
+        removeJournalAbbreviationsButtonGraphic.getStyleClass().add("icon");
+        removeJournalAbbreviationsButton.setGraphic(removeJournalAbbreviationsButtonGraphic);
+        ButtonBar.setButtonData(progressIndicator, ButtonData.LEFT);
+        ButtonBar.setButtonData(loadingLabel, ButtonData.LEFT);
+        ButtonBar.setButtonUniformSize(progressIndicator, false);
+        ButtonBar.setButtonUniformSize(loadingLabel, false);
+    }
+
+    private void setUpTable() {
+        journalAbbreviationsTable.setOnKeyPressed(event -> {
+            if ((event.getCode() == KeyCode.DELETE) && isEditableAndRemovable.get()) {
```

Comment:
Just notify the viewModel about the pressed key and let it decide what to do, i.e.
`if (event.getCode == KeyCode.DELETE) viewModel.deleteCurrentAbbreviation()` and the rest of the logic is in the viewModel.